File: objective-c/AsyncAwait/AsyncAwait_v1/AsyncAwait_v1/ViewControllers/ARootNavigationController.m
//
//  ARootNavigationController.m
//  AsyncAwait_v1
//
//  Created by Alexandr Kirilov on 21/12/2022.
//

#import "ARootNavigationController.h"

@interface ARootNavigationController ()

@end

@implementation ARootNavigationController

-(instancetype) init {
	
	self = [super init];
	if (self) {
		
	}
	return self;
}

-(void) loadView {
    
    [super loadView];
    [self.view setBackgroundColor:__COLOR_BLUE];
	
	[self.navigationBar setBarStyle:UIBarStyleBlack];
}

-(void)viewDidLoad {
    
    [super viewDidLoad];
}

@end

File: swift/AsyncAwait/AsyncAwait_v1/AsyncAwait_v1/ViewControllers/ARootNavigationController.swift
//
//  ARootNavigationController.swift
//  AsyncAwait_v1
//
//  Created by Alexandr Kirilov on 20/12/2022.
//

import UIKit

class ARootNavigationController: UINavigationController {


	override func loadView() {
		
		super.loadView();
		self.view.backgroundColor = __COLOR_GREEN_LIGHT;
		self.navigationBar.barStyle = .black;
	}
}
